web/crux/src/services/email-builder.service.ts

```diff
@@ -0,0 +1,51 @@
+import { Injectable } from '@nestjs/common'
+import { disassembleKratosRecoveryUrl } from 'src/domain/utils'
+import { EmailItem } from './email.service'
+
+type InviteTemaple = {
+  subject: string
+  text: string
+  html: string
+}
+
+const HOST = process.env.CRUX_UI_URL
+const sender = { email: process.env.FROM_EMAIL, name: process.env.FROM_NAME }
+
+@Injectable()
+export class EmailBuilder {
+  buildInviteEmail(email: string, teamName: string, teamId?: string, kratosRecoveryLink?: string): EmailItem | null {
+    if (!teamId && !kratosRecoveryLink) {
+      return null
+    }
+
+    const inviteTemplate = this.getInviteTemplate(teamName, teamId, kratosRecoveryLink)
+
+    const emailItem: EmailItem = {
+      sender: sender,
```

Comment:
I think we can stick with the SendGrid.MailDataRequired properties. (instead of sender and recipient use, from and to)

Reply:
After you don't need to implement a custom `EmailItem` we can use the SendGrid.MailData type. (for now its more readable than add some extra type)